common/utils/RandomGenUtils.php

```diff
@@ -39,10 +39,10 @@ public static function produceRandomString($length = 26, $alphabet = 'abcdefghij
             try {
                 $str .= $alphabet[random_int(0, $alphamax)];
             } catch (Error $e) {
-                error_log('OpenEMR Error : Encryption is not working because of random_int() Error: ' . $e->getMessage());
+                error_log('OpenEMR Error : Encryption is not working because of random_int() Error: ' . errorLogEscape($e->getMessage());
```

Comment:
need another parenthesis at end of this line (otherwise this will throw a parsing error)